internal/daysteps/daysteps.go

```diff
@@ -11,10 +18,61 @@ const (
 	mInKm = 1000
 )
 
+var (
+	errorInputData          = errors.New("the input data format does not meet the requirements")
+	errorCountOfStepsIsZero = errors.New("the count of steps cannot be zero")
+	errorDurationIsZero     = errors.New("duration cannot be zero")
+)
+
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+
+	splitData := strings.Split(data, ",")
+
+	if len(splitData) != 2 {
+		return 0, 0, fmt.Errorf("%w: %q", errorInputData, data)
+	}
+
+	countOfSteps, err := strconv.Atoi(splitData[0])
+
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше оборачивать ошибки
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```